backend/src/main/scala/cromwell/backend/standard/StandardInitializationActor.scala

```diff
@@ -0,0 +1,67 @@
+package cromwell.backend.standard
+
+import akka.actor.ActorRef
+import cromwell.backend.validation.RuntimeAttributesDefault
+import cromwell.backend.{BackendConfigurationDescriptor, BackendWorkflowDescriptor, BackendWorkflowInitializationActor}
+import cromwell.core.WorkflowOptions
+import wdl4s.TaskCall
+import wdl4s.values.WdlValue
+
+import scala.concurrent.Future
+import scala.util.Try
+
+trait StandardInitializationActorParams {
+  val workflowDescriptor: BackendWorkflowDescriptor
+  val calls: Set[TaskCall]
+  val serviceRegistryActor: ActorRef
+  val configurationDescriptor: BackendConfigurationDescriptor
+}
+
+case class DefaultInitializationActorParams
+(
+  workflowDescriptor: BackendWorkflowDescriptor,
+  calls: Set[TaskCall],
+  serviceRegistryActor: ActorRef,
+  configurationDescriptor: BackendConfigurationDescriptor
+) extends StandardInitializationActorParams
+
+trait StandardInitializationActor extends BackendWorkflowInitializationActor {
+  /** Standard set of parameters passed to the backend. */
+  def standardParams: StandardInitializationActorParams
+
+  override lazy val serviceRegistryActor: ActorRef = standardParams.serviceRegistryActor
+
+  override lazy val calls: Set[TaskCall] = standardParams.calls
+
+  def runtimeAttributesBuilder: StandardValidatedRuntimeAttributesBuilder =
+    StandardValidatedRuntimeAttributesBuilder.default
+
+  override protected def runtimeAttributeValidators: Map[String, (Option[WdlValue]) => Boolean] = {
+    runtimeAttributesBuilder.validatorMap
+  }
+
+  override protected def coerceDefaultRuntimeAttributes(options: WorkflowOptions): Try[Map[String, WdlValue]] = {
+    RuntimeAttributesDefault.workflowOptionsDefault(options, runtimeAttributesBuilder.coercionMap)
+  }
+
+  override def validate(): Future[Unit] = {
+    Future.fromTry(Try {
```

Comment:
Yep, it's all async-ified already. I don't like our thread queues needing to run runnables split into two (one for akka, one for scala Futures) for no real reason. As long as we're using `val ec: ExecutionContext = context.dispatcher`, I prefer:

|ec runnables queue|
|---|
|Akka `receive` launching a scala `Future`|
|The scala `Future` runnable|

Instead of:

|ec runnables queue|
|---|
|Actor `receive` just runs the runnable|

Further ToL: I'm fine measuring performance and moving blocking code to a separate thread pool.  Still in theory I'd prefer a leaf-actor do the heavy lifting, and not a scala Future.